src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -1250,7 +1250,7 @@ internal static string GetWindowsPowerShellModulePath()
             }
```

Comment:
@adityapatwardhan, your last commit had 2 failures in `PowerShell-CI-macos`
Test-Connection.Ping.Force IPv4 with implicit PingOptions
```powershell
Expected exactly '127.0.0.1', but got 10.79.0.23.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 83
83:             $result[0].Address | Should -BeExactly $realAddress
```
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '10.79.0.59', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 257
257:             $result[0].TargetAddress | Should -BeExactly $realAddress
```